src/modules/EnvironmentVariables/EnvironmentVariables/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,336 @@
+            }
+        }
+
+        [RelayCommand]
+        public void LoadEnvironmentVariables()
+        {
+            LoadDefaultVariables();
+            LoadProfiles();
+            PopulateAppliedVariables();
+        }
+
+        private void LoadProfiles()
+        {
+            try
+            {
+                var profiles = _environmentVariablesService.ReadProfiles();
+                foreach (var profile in profiles)
+                {
+                    profile.PropertyChanged += Profile_PropertyChanged;
+
+                    foreach (var variable in profile.Variables)
+                    {
+                        variable.ParentType = VariablesSetType.Profile;
+                    }
+                }
+
+                var appliedProfiles = profiles.Where(x => x.IsEnabled).ToList();
+                if (appliedProfiles.Count > 0)
+                {
+                    var appliedProfile = appliedProfiles.First();
+                    if (appliedProfile.IsCorrectlyApplied())
+                    {
+                        AppliedProfile = appliedProfile;
+                        IsStateModified = EnvironmentState.Unchanged;
+                    }
+                    else
+                    {
+                        IsStateModified = EnvironmentState.ChangedOnStartup;
+                        appliedProfile.IsEnabled = false;
+                    }
+                }
+
+                Profiles = new ObservableCollection<ProfileVariablesSet>(profiles);
+            }
+            catch (Exception ex)
+            {
+                // Show some error
+                Logger.LogError("Failed to load profiles.json file", ex);
+            }
+        }
+
+        private void PopulateAppliedVariables()
+        {
+            LoadDefaultVariables();
+
+            var variables = new List<Variable>();
+            if (AppliedProfile != null)
+            {
+                variables = variables.Concat(AppliedProfile.Variables.OrderBy(x => x.Name)).ToList();
+            }
+
+            variables = variables.Concat(UserDefaultSet.Variables.OrderBy(x => x.Name)).Concat(SystemDefaultSet.Variables.OrderBy(x => x.Name)).ToList();
+
+            // Handle PATH variable - add USER value to the end of the SYSTEM value
+            var userPath = variables.Where(x => x.Name.Equals("PATH", StringComparison.OrdinalIgnoreCase) && x.ParentType == VariablesSetType.User).FirstOrDefault();
+            var systemPath = variables.Where(x => x.Name.Equals("PATH", StringComparison.OrdinalIgnoreCase) && x.ParentType == VariablesSetType.System).FirstOrDefault();
+
+            if (!string.IsNullOrEmpty(userPath.Name) && !string.IsNullOrEmpty(systemPath.Name))
+            {
+                var clone = systemPath.Clone();
+                clone.Values += ";" + userPath.Values;
+                variables.Remove(userPath);
+                variables.Insert(variables.IndexOf(systemPath), clone);
```

Comment:
I think this should either be marked as user scoope or have it's own type as it contains `computer + user` value.

Reply:
hm, makes sense to make it obvious. not sure which one to go for though

Reply:
What about the path symbol. It's a special var and therefore it has special handling.

By the way, it makes a difference in zhe Windows env var ui if the name is `PATH` or `Path`. If it is `PATH` a special list dialogue opens on editing.

Reply:
> What about the path symbol. It's a special var and therefore it has special handling.

yeah, that one too. I'll see what to do there (for both path and duplicate vars)

> By the way, it makes a difference in zhe Windows env var ui if the name is PATH or Path. If it is PATH a special list dialogue opens on editing.

Will be added (for v2?). Trying to figure out the best UI for it.

Reply:
Some of these maybe E7EE EE57 EF58 F180 F6A9 ? For Path and duplicates? WDYT?

Reply:
For the path var I would use E8AC (Rename). And for duplicated vars I would use E8C8 (Copy) or E8C9 (Important). The icons for system and user looked good to me.